python/sdk/tests/test_build.py

```diff
@@ -326,17 +326,17 @@ def test_list_threat_hunts(self, unwrap):
         owners = set([r['account_id'] for r in res])
         assert {'prelude', pytest.account.headers['account']} >= owners
 
-        mine = [r for r in res if r['threat_hunt_id'] == pytest.expected_threat_hunt['threat_hunt_id']]
+        mine = [r for r in res if r['id'] == pytest.expected_threat_hunt['id']]
         assert 1 == len(mine)
         diffs = check_dict_items(pytest.expected_threat_hunt, mine[0])
         assert not diffs, json.dumps(diffs, indent=2)
 
     def test_update_threat_hunt(self, unwrap):
         pytest.expected_threat_hunt = unwrap(self.build.update_threat_hunt)(
             self.build,
-            threat_hunt_id=pytest.threat_hunt_id,
             name='updated threat hunt',
-            query='.*')
+            query='.*',
+            threat_hunt_id=pytest.threat_hunt_id)
```

Comment:
shouldn't this be `id`?

Reply:
all the other functions in our SDK (createTest/Threat, etc) name params this explicitly, so sticking with that